test/system/252-quadlet.bats

```diff
@@ -1212,9 +1212,9 @@ EOF
 
     echo "$_LOG_PROMPT journalctl -u $QUADLET_SERVICE_NAME"
     run -0 journalctl -eu $QUADLET_SERVICE_NAME
-    assert "$output" =~ "$port: bind: address already in use" "journal contains the real podman start error"
+    assert "$output" =~ ".*: address already in use" "journal contains the real podman start error"
```

Comment:
That change seems unrelated?  This is matching the podman error message which logically must still be the same.

Reply:
You're right. Fixed.